CMSgov/mint-app
# Easi 2616/user account migration aggregate task list sections

# EASI-2616 ,  EASI-2613, EASI-2614, EASI-2612, EASI-2617

## Changes and Description

Backend
- Migrated remaining task list sections to use the user table.
- Disable operational need needed trigger until that can be migrated as well

Front End
- Updated `ReadyForReview` component to deprecate query and `readyForReviewBy`
- Updated all queries that utilize `Ready for review` to now use `readyForReviewByUserAccount`
- Resolved existing issue with cache - was loading previous `Ready for review` status on first render
- Included `id` in queries where applicable
- Updated tests



## How to test this change
1. Run the app
2.  Ensure that you can update all task list sections


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
